### DISCUSSION TITLE
How to update an app manually?

### DISCUSSION THREAD

**Discussion by krelltunez** [nextcloud/all-in-one#3953] [Questions] [answered]
Hi, I'm trying to figure out how to (for example) update to Nextcloud Talk v18.0.1, but NextCloud AIO shows v17.1.4 as the latest available version.  Is there a way to do this?  Or, if using AIO, are you "stuck" with the versions of the apps that are included by default?

I could have sworn that my NextCloud Talk actually updated to v18 when it was released, but next time I went back to it that wasn't the case and it was "stuck" on v17.1.4.

If this is the way it is with AIO, that's fine... I just want to make sure I'm not missing something.

**Comment by szaimen** [comment, marked as answer]
> I just want to make sure I'm not missing something.

You are missing that Talk v18 is meant for Nextcloud 28. See https://github.com/nextcloud/all-in-one/discussions/2692
